bpf/kprobe_pwru.c

```diff
@@ -1,6 +1,6 @@
-// SPDX-License-Identifier: GPL-2.0-only
-/* Copyright (C) 2020-2021 Martynas Pumputis */
-/* Copyright (C) 2021 Authors of Cilium */
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Sorry, one more. I see this file has `char __license[] SEC("license") = "GPL";` so I'm not sure it's accurate to say Apache-2.0 in the SPDX header? Do we need to dual-license it?

Reply:
Ouch, good point!